src/util/vue-compat/index-2.js

```diff
@@ -17,6 +17,24 @@ export function getDefaultSlot(component) {
   return component.$slots.default;
 }
 
+export function renderToString(app) {
+  let _renderToString;
+  try {
+    _renderToString = require('vue-server-renderer/basic');
```

Comment:
should this also be eslint-disabled or is it always present?

Reply:
For now, we already have it as a dev dep, eslint doesn't complain about it.